packages/serve/index.ts

```diff
@@ -170,7 +170,7 @@ function serve() {
 	}
 }
 
-export = {
+export {
```

Comment:
we're using these, so needs to be grouped export

Reply:
refactor opportunity here

Reply:
What do you mean? There are no breaking changes here, its simply adding the default export onto the object of existing exported functions.